server/services/zwave/lib/comClass/alarmCommandClass.js

```diff
@@ -0,0 +1,41 @@
+const { COMMAND_CLASSES, INDEXES } = require('../constants');
+const DefaultCommandClass = require('./defaultCommandClass');
```

Comment:
It's a little weird to have a capitalized require that is not a class.

```
const defaultCommandClass = require('./defaultCommandClass');
```

Reply:
Fixed